profiles/GurjeetSinghVirdee.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/73753957/147848925-4655ce63-2950-4deb-9931-e8140909bf74.png",
```

Sorry if I reverted this, Krish, didn't see this properly. 😅
```suggestion
  "image": "https://user-images.githubusercontent.com/91655303/147851903-7af25096-b607-42a2-bbfb-e7e77c53ecb6.png",
```